src/components/BulkEnrollmentPage/stepper/BulkEnrollmentSubmit.jsx

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+import { logError } from '@edx/frontend-platform/logging';
+import {
+  FINAL_BUTTON_TEST_ID,
+  FINAL_BUTTON_TEXT,
+  NOTIFY_CHECKBOX_TEST_ID,
+  CUSTOMER_SUPPORT_HYPERLINK_TEST_ID,
+  ALERT_MODAL_TITLE_TEXT,
+  ALERT_MODAL_BODY_TEXT,
+  SUPPORT_HYPERLINK_TEXT,
+} from './constants';
+import LicenseManagerApiService from '../../../data/services/LicenseManagerAPIService';
+import { BulkEnrollContext } from '../BulkEnrollmentContext';
+import { ToastsContext } from '../../Toasts';
+import { clearSelectionAction } from '../data/actions';
+
+export const BulkEnrollmentAlertModal = ({ isOpen, toggleClose, enterpriseSlug }) => (
+  <AlertModal
+    title={ALERT_MODAL_TITLE_TEXT}
+    isOpen={isOpen}
+    onClose={toggleClose}
+    footerNode={(
+      <ActionRow>
+        <Button variant="primary" onClick={toggleClose}>OK</Button>
+      </ActionRow>
+    )}
+  >
+    <p>
+      {ALERT_MODAL_BODY_TEXT}
+      <Hyperlink
+        destination={`/${enterpriseSlug}/admin/support`}
+        target="_blank"
+        rel="noopener noreferrer"
+        data-testid={CUSTOMER_SUPPORT_HYPERLINK_TEST_ID}
+      >
+        {SUPPORT_HYPERLINK_TEXT}
+      </Hyperlink>
+    </p>
+  </AlertModal>
+);
+
+const BulkEnrollmentSubmit = ({ enterpriseId, enterpriseSlug, returnToInitialStep }) => {
+  const [loading, setLoading] = useState(false);
+  const [checked, setChecked] = useState(true);
+  const handleChange = e => setChecked(e.target.checked);
+
+  const {
+    emails: [selectedEmails, emailsDispatch],
+    courses: [selectedCourses, coursesDispatch],
+  } = useContext(BulkEnrollContext);
+  const { addToast } = useContext(ToastsContext);
+
+  let courses;
+  if (selectedCourses.original?.advertised_course_run) {
+    courses = selectedCourses?.map(
+      ({ selectedRow }) => selectedRow.original?.advertised_course_run?.key,
+    );
+  } else {
+    courses = selectedCourses.map(({ id }) => id);
+  }
+  const emails = selectedEmails.map(({ values }) => values.userEmail);
+  const [isOpen, toggleErrorModalOpen, toggleErrorModalClose] = useToggle();
+  const hasSelectedCoursesAndEmails = selectedEmails.length > 0 && selectedCourses.length > 0;
+
+  const submitBulkEnrollment = () => {
+    setLoading(true);
+    const options = {
+      emails,
+      course_run_keys: courses,
+      notify: checked,
+    };
+
+    return LicenseManagerApiService.licenseBulkEnroll(
+      enterpriseId,
+      options,
+    ).then(() => {
+      coursesDispatch(clearSelectionAction());
+      emailsDispatch(clearSelectionAction());
+      addToast(`${selectedEmails.length} learners have been enrolled.`);
+      returnToInitialStep();
+    }).catch((err) => {
+      logError(err);
+      toggleErrorModalOpen();
+      setLoading(false);
+    });
+  };
+
+  return (
+    <>
+      <BulkEnrollmentAlertModal
+        enterpriseSlug={enterpriseSlug}
+        toggleClose={toggleErrorModalClose}
+        isOpen={isOpen}
+      />
+      <Form.Checkbox
```

Comment:
We should discuss, probably with @wittjeff if this should be wrapped in a `<Form>` component. I suspect yes.

Reply:
Talked with Jeff, I think we're ok for now! 👍